### DISCUSSION TITLE
getStaticProps fallBack: true

### DISCUSSION THREAD

**Discussion by ReactNewbie-J** [vercel/next.js#22752] [Help]
Hi, I'm new to NextJs, I like the concept of the dynamic routing and I'm using the Static Html Export for production, I don't really get why `fallback: true is not supported when using next export`, I can see [here](https://nextjs.org/docs/basic-features/data-fetching#when-is-fallback-true-useful) that I can still use the `fallBack: true` in static pages, but how do I do that? how do I export static version of my app without using `next export` and able to use `fallBack: true` on static pages.

**Comment by leerob** [comment]
The reason fallback isn't supported with `next export` is because it requires a server to "fallback" to. This statically generates your page at request time. With `next export`, you have no server.

If you want to use `fallback`, you should use `next start` and deploy to a platform that supports it 👍
